changelog/24297.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+logging: Vault server, Agent and Proxy not honor log file value and only timestamp on rotation.
+```
```

Comment:
```suggestion
```release-note:change
logging: Vault server, Agent and Proxy now honor log file value and only add a timestamp on rotation.
```
```